ArcGISRuntimeSDKQt_QMLSamples/EditData/DeleteFeaturesFeatureService/DeleteFeaturesFeatureService.qml

```diff
@@ -143,71 +142,26 @@ Rectangle {
                 }
             }
         }
-    }
-
-    // map callout window
-    Rectangle {
-        id: callout
-        width: row.width + (10 * scaleFactor) // add 10 for padding
-        height: 40 * scaleFactor
-        radius: 5
-        border {
-            color: "lightgrey"
-            width: .5
-        }
-        visible: false
 
-        MouseArea {
-            anchors.fill: parent
-            onClicked: mouse.accepted = true
+        calloutData {
+            location: calloutLocation
+            // We use the HTMl to bold, increase the pt size, and centre-align the the damageType title.
```

Comment:
- `HTMl` -> `HTML`
- `centre` -> `center` (sorry, I know both are correct, but we want to be consistent throughout our doc/samples, so let's stick with US versions where possible)